gemfiles/rails_4.gemfile

```diff
@@ -0,0 +1,51 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+ruby "2.3.0"
+
+gem "administrate-field-image"
+gem "delayed_job_active_record"
+gem "high_voltage"
+gem "markdown-rails"
+gem "pg"
+gem "redcarpet"
+gem "unicorn"
+gem "rails", "~> 4.2"
+gem "rspec-rails", "~> 3.4.2"
+
+group :development do
+  gem "web-console", ">= 2.1.3"
+end
+
+group :development, :test do
+  gem "appraisal"
+  gem "awesome_print"
+  gem "bundler-audit", :require => false
+  gem "byebug"
+  gem "dotenv-rails"
+  gem "factory_girl_rails"
+  gem "faker"
+  gem "i18n-tasks"
+  gem "pry-rails"
+  gem "fuubar", "~> 2.1.0.beta2"
+end
+
+group :test do
+  gem "ammeter"
+  gem "database_cleaner"
+  gem "formulaic"
+  gem "launchy"
+  gem "poltergeist"
+  gem "shoulda-matchers", "~> 2.8.0", :require => false
+  gem "timecop"
+  gem "webmock"
+end
+
+group :staging, :production do
+  gem "rack-timeout"
+  gem "rails_stdout_logging"
+  gem "uglifier"
+end
+
+gemspec :path => "../"
```

Comment:
They are autogenerated, but we are taught to commit our gemfiles to share expectations of compatibility. We also want to version to manage potential regressions.